samples/tfm/tfm_psa_test/sample.yaml

```diff
@@ -0,0 +1,25 @@
+sample:
+common:
+    tags: tfm
+    platform_allow: nrf5340pdk_nrf5340_cpuappns nrf9160dk_nrf9160ns
+    harness: console
+    harness_config:
+      type: multi_line
+      regex:
+        - "\\*\\*\\*\\*\\* PSA Architecture Test Suite - Version 1.0 \\*\\*\\*\\*\\*"
+        - "TOTAL FAILED    : 0"
+sample:
+  name: "TFM PSA Test"
+
+tests:
+  sample.tfm.psa_storage_test:
+    tags: tfm
+    extra_args: "CONFIG_TFM_PSA_TEST_STORAGE=y"
+    timeout: 100
+  # sample.tfm.psa_crypto_test:
```

Comment:
Work in progress or should this be removed?

Reply:
Work in progress. I'd like to keep it until these suites work (depends on @frkv's work)